apps/console/src/features/identity-providers/data/identity-provider-templates/templates/saml-identity-provider/saml-idp-wizard-file-based-help.tsx

```diff
@@ -39,8 +40,48 @@ const SAMLIdPWizardFileBasedHelp: FunctionComponent<Props> = (props: Props): Rea
     const { [ "data-testid" ]: testId } = props;
     const config: ConfigReducerStateInterface = useSelector((state: AppState) => state.config);
 
+    const { t } = useTranslation();
+    const { getLink } = useDocumentation();
+
     return (
         <div data-testid={ testId }>
+            <Message info>
+                <p>
+                    <Trans
+                        i18nKey={
+                            "console:develop.features.authenticationProvider.templates.enterprise.saml." +
+                            "preRequisites.configureRedirectURL"
+                        }
+                    >
+                        Use the following URL as the <strong>Authorized Redirect URI</strong>.
+                    </Trans>
+                    <CopyInputField
+                        className="copy-input-dark spaced"
+                        value={ config?.deployment?.serverHost + "/commonauth" }
+                    />
+                    <Icon name="info circle" />
+                    {
+                        t("console:develop.features.authenticationProvider.templates.enterprise.saml." +
+                            "preRequisites.hint", {
+                            productName: config.ui.productName
+                        })
+                    }
+                    <br/>
+                    <br/>
+                    { getLink("develop.connections.newConnection.enterprise.samlLearnMore") === undefined ?
+                        null :
+                        <a
```

Comment:
Cant we use the `DocumentationLink` component here.

Reply:
Fixed with 294f6a756cf6d9da34aaa39c0cc0835f4bf065ad